aten/src/ATen/native/cudnn/MHA.cpp

```diff
@@ -614,9 +655,44 @@ void run_cudnn_SDP_bprop(
     Tensor& dV,
     const Tensor& dropoutseed,
     const Tensor& dropoutoffset) {
+  Tensor dO_ = dO;
+  if (!dO.strides()[dO.strides().size() - 1]) {
+    TORCH_WARN(
+        "cuDNN SDPA backward got an innermost stride of 0 in grad_out, which is unsupported."
```

Comment:
Did the cudnn_frontend upgrade fix this?

Reply:
Innermost strides of 0 are presumably unsupported, but we could maybe check if some of the outer strides are 0